app/routes/app/workspaces/start/$workspaceExternalId.tsx

```diff
@@ -33,7 +33,12 @@ export const action = async ({ context: { app, db, req, user } }: ActionArgs) =>
   if (workspace.userId !== unwrap(user).id) {
     throw new HttpError(StatusCodes.FORBIDDEN, "Workspace does not belong to user");
   }
-  if (workspace.status !== WorkspaceStatus.WORKSPACE_STATUS_STOPPED) {
+  if (
+    ![
+      WorkspaceStatus.WORKSPACE_STATUS_STOPPED,
```

Comment:
What about failed to start?

Reply:
There's no such status. If a VM fails to start it goes into the `WORKSPACE_STATUS_STOPPED_WITH_ERROR`.